openedx/core/djangoapps/discussion_common/comment_client/thread.py

```diff
@@ -176,8 +179,6 @@ def flagAbuse(self, user, voteable):
     def unFlagAbuse(self, user, voteable, removeAll):
         if voteable.type == 'thread':
             url = _url_for_unflag_abuse_thread(voteable.id)
-        elif voteable.type == 'comment':
-            url = _url_for_unflag_comment(voteable.id)
```

Comment:
Why was this removed?

Reply:
Same as `_url_for_flag_comment`, `_url_for_unflag_comment` isn't defined anywhere. 

I'd love to run coverage.py in production and find all dead code in our codebase!